src/app/components/pages/Equality.tsx

```diff
@@ -382,20 +381,6 @@ const Equality = withRouter(
               <Col md={{ size: 8, offset: 2 }} className="py-4 inequality-results">
                 <h4>LaTeX</h4>
                 <pre>${currentAttemptValue?.result?.tex}$</pre>
-                {editorMode === "chemistry" && (
-                  <>
-                    <h4>MhChem</h4>
-                    <pre>{currentAttemptValue?.result?.mhchem}</pre>
-                  </>
-                )}
-                {editorMode !== "chemistry" && (
```

Comment:
Is this intended? The data inside this block will not show now, even if `editorMode` is `maths` or `logic`.